modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -107,9 +107,11 @@ Once this is successfully configured, you can apply the OpenID Connect OAuth Tok
 [NOTE]
 For Okta, the Okta admin needs to assign the dynamically generated clients to a user or a group of users in order for them to receive access tokens by sending over the Client ID and Client Secret.
 
-== Grant types
+== Displaying grant types in API portal
```

Comment:
```suggestion
== Displaying Grant Types in API Portal
```

Reply:
in an API portal? in the API portal?

Reply:
@sarathecoubian Should we call it API Portal?  Or is this more specifically Exchange? Not sure if there is a specific reason this doc uses "api portal" rather than "exchange".  Thoughts?

Reply:
If it’s referring to Exchange, it should be Anypoint Exchange. Not sure what the API portal is in this context.

Reply:
@sebastiankorol Any insight into why this doc says "api portal" and not "Exchange" specifically?  Is there some history there?

Reply:
The API Portal is a section of Exchange.

Reply:
https://www.mulesoft.com/resources/api/what-is-an-api-portal